packages/host/src/AppBar.tsx

```diff
@@ -129,12 +64,14 @@ const AppBar: React.FC = () => {
           <Button
             icon={<Book />}
             labelKey="button.docs"
-            onClick={() => (location.href = 'https://docs.msupply.foundation')}
+            onClick={() => (location.href = ExternalURL.PublicDocs)}
           />
           <LanguageMenu />
         </ButtonContainer>
-      </StyledToolbar>
+      </Toolbar>
+      <ButtonContainer ref={appBarExtraRef} />
```

Comment:
the `<ButtonContainer />` could do with a new name to avoid confusion

Reply:
Will do - will go and add some things into it and do it in the next pr